### ISSUE TITLE
HTML Links in notes don't open

### ISSUE THREAD

**Issue by zeigerpuppy** [jparyani/laverna#15]
when clicking on an html link in a note, Laverna doesn't open the link.
right-click and "open in new tab" is required.

**Comment by funwhilelost** [comment]
This is a Sandstorm security feature afaik.

**Comment by ocdtrekkie** [comment]
Yeah, basically the link just needs to have target="_blank" added to it.

**Comment by zeigerpuppy** [comment]
OK, thanks for the tip.  It's a bit disconcerting to users who are creating documents with Laverna, but I guess there must be some security benefit to it.

**Comment by funwhilelost** [comment]
Yeah, the grain is in an iframe so if you followed a link without `target="_blank"` you would "lose" your grain.
